tests/test_chain_service.py

```
@@ -115,6 +124,38 @@ def test_chain_orchestration_progress_and_intermediate(chain_env, tmp_path):
     assert not work.exists()
 
 
+def test_chain_tags_final_chd_via_post_convert(chain_env, tmp_path, monkeypatch):
+    """The chain drives the final step's ``post_convert`` with the intermediate
+    source and the final CHD — the same disc-ID path a direct ``createdvd`` job
+    takes. This is what replaces the old bespoke ``_embed_disc_id`` (#181)."""
+    calls, work, install = chain_env
```

Ruff (0.15.18)

[warning] Unpacked variable `calls` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)